DependencyInjection/Compiler/DbConnectionPass.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of TheCadien/SuluImportExportBundle.
+ *
+ * (c) Oliver Kossin
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace TheCadien\Bundle\SuluImportExportBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * This compiler pass is responsible for fetching the connection params
+ * of the DBAL and passes them to our export and import service.
+ */
+class DbConnectionPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        $connectionName = $container->getParameter('sulu_import_export_bundle.dbal_connection');
+
+        if (!$connectionName) {
+            $connectionName = $container->getParameter('doctrine.default_connection');
+        }
+
+        $connections = $container->getParameter('doctrine.connections');
+
+        if (!isset($connections[$connectionName])) {
+            throw new \InvalidArgumentException(sprintf('There is no doctrine connection configured which is named "%s". Available names are "%s".', $connectionName, implode('", "', array_keys($connections))));
+        }
+
+        $connectionId = $connections[$connectionName];
+        $connectionParams = $container->getDefinition($connectionId)->getArgument(0);
+
+        $exportService = $container->getDefinition('sulu.service.export');
+        $exportService->setArgument('$databaseParams', $connectionParams);
+
+        $importService = $container->getDefinition('sulu.service.import');
+        $importService->setArgument('$databaseParams', $connectionParams);
```

Comment:
It would be nice to have this in the Extension instead, but I think going with this CompilerPass is the better choice, because we can be certain that the Doctrine configuration is already there. If I remember correctly, using the Extension for getting the config will require putting the bundles in the right order in the bundles.php or some additional handling for priority, which is quite annoying.